packages/lib-sourcify/src/lib/logger.ts

```diff
@@ -0,0 +1,58 @@
+export interface ILogger {
+  logLevel: number;
+  log: (level: number, message: string) => void;
+  setLevel: (level: number) => void;
+}
+
+// Default logger behavior
+export const DefaultLogger: ILogger = {
+  logLevel: 1,
```

Comment:
Isn't default 3?

Reply:
From a quick search most libraries implement the warn level